clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10188,7 +10188,7 @@ void SPIRVTranslator::ConstructJob(Compilation &C, const JobAction &JA,
     std::string DefaultExtArg =
         ",+SPV_EXT_shader_atomic_float_add,+SPV_EXT_shader_atomic_float_min_max"
         ",+SPV_KHR_no_integer_wrap_decoration,+SPV_KHR_float_controls"
-        ",+SPV_KHR_expect_assume,+SPV_KHR_linkonce_odr";
+        ",+SPV_KHR_expect_assume,+SPV_KHR_linkonce_odr,+SPV_KHR_bit_instructions";
```

Comment:
Can you also update `clang/test/Driver/sycl-spirv-ext.c`?

Reply:
@mdtoguchi Stefan is out at the moment and I don't have write permissions, could you make that update ?

Reply:
Hey it worked!  done :)

Reply:
@mdtoguchi perfect! if it's approved and ready-to-be-merged, can you merge it ? I don't have write permissions for this either.